test/corker/reports/monthly_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule Corker.Reports.MonthlyTest do
+  use Corker.DataCase, async: true
+
+  alias Corker.Reports.Monthly
+
+  describe "generate/0" do
+    test "returns the leaderboards for this month" do
+      top_sender = insert(:user)
+      top_receiver = insert(:user)
+      avg_user = insert(:user)
+      insert(:high_five, receiver_id: top_receiver.id, sender_id: top_sender.id)
+      insert(:high_five, receiver_id: top_receiver.id, sender_id: avg_user.id)
+      insert(:high_five, receiver_id: avg_user.id, sender_id: top_sender.id)
+
+      {sender_stats, receiver_stats} = Monthly.generate()
+
+      # sender leaderboard
```

Comment:
Really need those comments?

Reply:
Uh, no, these were a result from copy/pasting from the original test. mb